build.zig.zon

```diff
@@ -1,15 +1,17 @@
 .{
-    .name = "zig-toxcore-c",
+    .name = "c-toxcore-zig",
```

Comment:
Why the rename?

Reply:
oh, the original repository name at nodecum is c-toxcore-zig...
Should I rename the original one? This could mess things up or?

Reply:
I mean how would github behave if I rename nodecum/c-toxcore-zig which is the origin of the fork of TokTok/zig-toxcore-c.